packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -150,10 +140,6 @@ export class PhoenixObjects {
       : CoordinateHelper.etaToTheta(eta);
     // Jet energy parameter can either be 'energy' or 'et'
     let length = (jetParams.energy ? jetParams.energy : jetParams.et) * 0.2;
-    // Ugh - We don't want the Jets to go out of the event display
```

Comment:
Is this fine for now? I was thinking of normalizing the energy in a certain range. The differences should then be apparent and Jets won't scale more than the event display.